packages/candy_editor/src/assets/formTypes/FloatForm/converters.ts

```diff
@@ -0,0 +1,9 @@
+import { CandyFloat } from '../../../types';
+
+export function convertToCandyFloat(typedValue: string): CandyFloat | undefined {
+    if (/^[-+]?[0-9]*\.?[0-9]+([eE][-+]?[0-9]+)?$/.test(typedValue)) {
```

Comment:
Needs a range check.

Reply:
https://origyn.atlassian.net/jira/software/projects/ENG/boards/37/roadmap?selectedIssue=ENG-1264

Reply:
Alessandro to create a bug and work on this later.